### PR TITLE
0.2.1

### PR DESCRIPTION
**Pull request by jtchitty**
Some improvements in preparation for the 0.2.1 release of mangOH Yellow.

### CODE REVIEW

File: mangOH-yellow-Manifest.json [hunk 1]
```diff
@@ -4,25 +4,25 @@
     "info": {
         "name": "mangOH-%MANGOH_BOARD%-%TARGET%",
         "version": "%VERSION%",
-        "description": "SDK for mangOH %MANGOH_BOARD% with %TARGET%",
+        "description": "SDK for mangOH %MANGOH_BOARD% with %TARGET%, based on Legato version %LEGATO_VERSION% and %TARGET% release %MODEM_RELEASE_VERSION%",
         "master": true,
         "requires": [
             "swi-license_1.1",
```
Comment (dpfrey) [inline mangOH-yellow-Manifest.json]: is this supposed to be 1.2 now?

File: Makefile [hunk 2]
```diff
@@ -185,12 +185,18 @@ $(LEGATO_SOURCES_FETCHED):
...
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/37/49737/1 && \
 		git cherry-pick FETCH_HEAD && \
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/73/49773/1 && \
 		git cherry-pick FETCH_HEAD && \
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/57/50657/1 && \
+		git cherry-pick FETCH_HEAD && \
+		git fetch ssh://gerrit.legato:29418/Legato refs/changes/68/50868/5 && \
+		git cherry-pick FETCH_HEAD && \
+		cd apps/platformServices/airVantageConnector && \
+		git fetch ssh://gerrit.legato:29418/Legato/Service/AirVantageConnector refs/changes/69/50869/9 && \
```
Comment (dpfrey) [inline Makefile:199]: Do we have confirmation from the Octave team that these are the only additional patched that we need?

**Review by dpfrey — COMMENTED** [review]
just a couple of little questions

File: mangOH-yellow-Manifest.json (hunk 1 above)
Comment (jtchitty) [reply mangOH-yellow-Manifest.json]: Yeah, I fixed that in a later commit.  You're looking at an outdated version of the code.

File: Makefile (hunk 2 above)
Comment (jtchitty) [reply Makefile:199]: No.  Actually, we know they bring in a bunch more.  The list has been changing rapidly lately, as they prepare for their GA release.

**Review by dpfrey — APPROVED** [review]

**Pull request merged**